Charts/Classes/Renderers/CandleStickChartRenderer.swift

```diff
@@ -43,7 +43,7 @@ public class CandleStickChartRenderer: LineScatterCandleRadarChartRenderer
         }
     }
     
-    private var _shadowPoints = [CGPoint](repeating: CGPoint(), count: 4)
+    private var _shadowPoints = [CGPoint](repeating: CGPoint(), count: 2)
```

Comment:
Yea give the conversion tool a try then you will understand. Anyways in this instance its simply a mistake. I can submit another commit with the fix once we go over the rest, to make sure there aren't any other mistakes.